buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -89,6 +90,12 @@ object Libraries {
     const val scarletLifecycle =  "com.tinder.scarlet:lifecycle-android:${Versions.scarlet}"
     const val scarletGson      =  "com.tinder.scarlet:message-adapter-gson:${Versions.scarlet}"
     const val fragment         =  "androidx.fragment:fragment:${Versions.fragment}"
+    const val composeUi        =  "androidx.compose.ui:ui:${Versions.compose}"
+    const val composeMaterial  =  "androidx.compose.material:material:${Versions.compose}"
+    const val composeTooling   =  "androidx.compose.ui:ui-tooling:${Versions.compose}"
+    const val composeActivity  =  "androidx.activity:activity-compose:1.4.0"
+    const val composeNavigation=  "androidx.navigation:navigation-compose:2.4.0-beta02"
```

Comment:
I think the convention is to keep all version numbers separate in the `object Versions` 🤔

Reply:
good point

Reply:
Yes, that's because some versions are applied to multiple dependencies, like `kotlinx-coroutines-core` and `kotlinx-coroutines-android`. So having a single version declared changes the version to all the related dependencies.

The idea is having everything under `Versions` so we have everything in the same place.